frontend/src/app/containers/ExperimentPage/index.js

```diff
@@ -609,7 +609,7 @@ export class ExperimentDetail extends React.Component {
   }
 
   renderMaximumVersionNotice(title, hasAddon, max_release) {
```

Comment:
You'll need to add graduated as a parameter to renderMaximumVersionNotice, and pass it where renderMaximumVersionNotice is called.

Reply:
👍